integration/test_flow_integration.py

```diff
@@ -49,3 +59,18 @@ def test_join_with_http():
     controller.terminate()
     termination_result = controller.await_termination()
     assert termination_result == 200 * 7
+
+
+def test_write_to_v3io_stream():
+    stream_path = f'bigdata/test_write_to_v3io_stream/{int(time.time_ns() / 1000)}/'
+    asyncio.run(SetupStream().setup(stream_path))
+    controller = build_flow([
+        Source(),
+        Map(lambda x: str(x)),
+        WriteToV3IOStream(stream_path, partition_func=lambda event: str(int(event.element) % 2))
+    ]).run()
+    for i in range(10):
+        controller.emit(i)
+
+    controller.terminate()
```

Comment:
actually assert that values were written, using v3io-py/nginx or whatever

Reply:
Yeah... Not excited about that.